charts/scrutiny/charts/collector/templates/daemonset.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
```

Comment:
I'd love to use Kubernetes's built in cron-scheduling, instead of my kludgy cron-in-docker solution if we can. 
I've used https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/ in the past, but I'm not sure if it has the capabilities to force a container on every node. Is that something you're familiar with?

Reply:
I completely agree, I do not particularly like the cron-in-docker thing. Unfortunately as far as I know, there is no reliable way to run a CronJob on all nodes :(
I've seen hacks here and there, like forcing a number of replicas == number of nodes and setting antiaffinity rules, but I don't think that's very reliable.

An alternative to the cron in docker could be a simpler image that just does:
```
#!/bin/sh
while [ true ]; do
    /opt/scrutiny/bin/something something
    sleep 6h
done
```

This script is simple enough to inline in the K8s manifest. WDYT?